.gitignore

```diff
@@ -113,3 +113,6 @@ win-64
 # generated coffeescript
 /bokehjs/test/common/generated_defaults/models_defaults.coffee
 /bokehjs/test/common/generated_defaults/widgets_defaults.coffee
+
+*.styl
+*.png
```

Comment:
Yeah, fallout from my attempts to figure out how to use git effectively on windows. This wasn't even supposed to be committed, it was auto-added by the github UI 😐